test/datasets/test_wikitexts.py

```diff
@@ -55,19 +57,19 @@ class TestWikiTexts(TempDirMixin, TorchtextTestCase):
     root_dir = None
     samples = []
 
-    @classmethod
+    @ classmethod
```

Comment:
nit: should we remove this space? same with the rest of the `@classmethod` and `@nested_params`  decorators

Reply:
hmm,  must be by mistake..already fixed :)